js/web/lib/wasm/wasm-core-impl.ts

```diff
@@ -483,7 +474,7 @@ export const run = async(
     }
 
     if (!BUILD_DEFS.DISABLE_WEBGPU && ioBindingState) {
-      if (!graphCaptureEnabled || (graphCaptureEnabled && !inputOutputBounded)) {
+      if (!enableGraphCapture || (enableGraphCapture && !inputOutputBounded)) {
```

Comment:
is this equivalent to (!enableGraphCapture || !inputOutputBound) ?

Reply:
After adding https://github.com/microsoft/onnxruntime/pull/18989/commits/dff25faa81bba7d296348e1b6bc41588cd471314#diff-d3f7d7b2ba7b5644ee5e29b09d15ebf104126b028bf01a53e61ca4597977b8c2R629-R631 when clear bound outputs, it's equivalent.